src/args.rs

```diff
@@ -117,6 +117,10 @@ struct CliArgs {
     /// Shown instead of host in page title and heading
     #[structopt(short = "t", long = "title")]
     title: Option<String>,
+
+    /// Custom header from user
+    #[structopt(long = "header", parse(try_from_str = parse_header))]
+    header: Option<Header>,
```

Comment:
Wouldn't it be cool to be able to set multiple headers here? So something like `Vec<Header>` might be nice I think. :)